analyzers/tests/SonarAnalyzer.UnitTest/SymbolicExecution/Roslyn/RoslynSymbolicExecutionTest.ExceptionCandidate.cs

```diff
@@ -512,5 +514,29 @@ public void ExceptionCandidate_MethodReference_Static()
             validator.ValidateTagOrder("BeforeTry", "InTry", "AfterCatch");
             validator.ExitStates.Should().HaveCount(1).And.ContainSingle(x => HasNoException(x));
         }
+
+        [TestMethod]
+        public void ExceptionCandidate_End()
+        {
+            const string code = @"
+Module Program
+    Sub Main(args As String())
+        Dim tag = ""BeforeTry""
+        Try
+            tag = ""InTry""
+            END
+        Catch
+            tag = ""InCatch""
+        End Try
+        tag = ""AfterCatch""
+    End Sub
+End Module";
+
+            var validator = new SETestContext(code, AnalyzerLanguage.VisualBasic, additionalChecks: Array.Empty<SymbolicCheck>(), outputKind: OutputKind.ConsoleApplication).Validator;
```

Comment:
I would not name this.
```suggestion
            var validator = new SETestContext(code, AnalyzerLanguage.VisualBasic, Array.Empty<SymbolicCheck>(), outputKind: OutputKind.ConsoleApplication).Validator;
```